src/local-pv-monitor/cmd/main.go

```diff
@@ -20,24 +20,33 @@ import (
 	"flag"
 	"os"
 
+	lvmonitor "github.com/caicloud/kube-storage-monitor/src/local-pv-monitor/pkg/monitor"
```

Comment:
what's the reason for this layout?  `src/local-pv-monitor/cmd/main.go`.

ideally, we should just use `cmd/main.go`; here is our project template, FYI

```
├── .github
│   ├── ISSUE_TEMPLATE
│   └── PULL_REQUEST_TEMPLATE
├── .gitignore
├── .pre-commit-config.yaml
├── CHANGELOG.md
├── Makefile
├── CODEOWNERS
├── README.md
├── bin
│   ├── admin
│   └── controller
├── cmd
│   ├── admin
│   │   └── admin.go
│   └── controller
│       └── controller.go
├── build
│   ├── admin
│   │   ├── Dockerfile
│   └── controller
│       ├── Dockerfile
├── docs
│   └── README.md
├── hack
│   ├── README.md
│   ├── deployment.yaml
│   └── script.sh
├── pkg
│   ├── utils
│   │   └── net
│   │       └── net.go
│   └── version
│       └── version.go
├── test
│   └── README.md
│   └── test_make.sh
├── third_party
│   └── README.md
└── vendor
    └── README.md
```

A brief description of the layout:

* `.github` has two template files for creating PR and issue. Please see the files for more details.
* `.gitignore` varies per project, but all projects need to ignore `bin` directory.
* `.pre-commit-config.yaml` is for configuring [pre-commit](https://pre-commit.com/).
* `Makefile` is used to build the project.
* `CHANGELOG.md` contains auto-generated changelog information.
* `CODEOWNERS` contains owners of the project.
* `README.md` is a detailed description of the project.
* `bin` is to hold build outputs.
* `cmd` contains main packages, each subdirecoty of `cmd` is a main package.
* `build` contains scripts, yaml files, dockerfiles, etc, to build and package the project.
* `docs` for project documentations.
* `hack` contains scripts used to manage this repository, e.g. codegen, installation, verification, etc.
* `pkg` places most of project business logic.
* `test` holds all tests (except unit tests), e.g. integration, e2e tests.
* `third_party` for all third party libraries and tools, e.g. swagger ui, protocol buf, etc.
* `vendor` contains all vendored code.